update test data to realistic studies

magland opened the pull request.

magland committed:
initial commit

jsoules reviewed (commented):
Looks generally good; two things:

a) Would you mind tweaking the test we have [https://github.com/jsoules/othismos/blob/main/src/components/ExpandingHeatmapTable.test.tsx] as well, so that it'll continue to pass? (It was looking for specific text in the table cells)
b) What's the standard on when we use line-end semicolons? I thought we were leaving them out generally, but they're used here after the import statement and the various lists.

magland committed:
add .prettierrc update .test.tsx

magland commented:
Yes, I prefer excluding trailing semicolons. Codesandbox autoformatted. I added a .prettierrc that controls the auto formatting, and now the semicolons are gone. I also updated the test as requested. Could you check to see if it passes? @jsoules

jsoules commented:
Looks good to me (and test passes). Merge away (or I can do it if you prefer).
